analyses/transcriptomic-dimension-reduction/04-explore-sequencing-center-effects.Rmd

```diff
@@ -0,0 +1,311 @@
+  scale_color_manual(values = c("grey70", "black", "black")) +
+  scale_alpha_manual(values = c(0.4, 0.8, 0.8)) +
+  ggpubr::theme_pubr()
+```
+
+
+### Broad histology UMAP
+
+We can make this finer-grained by exploring the distributions of broad histologies within this UMAP.
+Shapes denote the sequencing center, and colors denote the broad histology.
+
+```{r}
+ggplot(stranded_df) + 
+  aes(x = UMAP1, y = UMAP2, 
+      shape = seq_center, 
+      fill = broad_histology_display, 
+      color = seq_center, 
+      alpha = seq_center) + 
+  geom_point() + 
+  scale_shape_manual(values = c(21, 22, 24)) + 
+  scale_fill_manual(values  = pal_bh) +
+  scale_color_manual(values = c("grey70", "black", "black")) +
+  scale_alpha_manual(values = c(0.4, 0.6, 0.6)) +
+  ggpubr::theme_pubr() + 
+  guides(fill = FALSE, alpha = FALSE)
+```
+
+
+We can also make this plot only for broad histologies which were processed at >1 center:
+
+```{r}
+# first, which histologies are shared, aka processed at >1 center?
+shared_bh <- multiple_centers_bh %>%
+  dplyr::filter(RNA_library == "stranded") %>%
+  dplyr::distinct(broad_histology_display) %>%
+  dplyr::pull(broad_histology_display)
+
+
+stranded_df %>%
+  dplyr::filter(broad_histology_display %in% shared_bh) %>%
+  ggplot() + 
+  aes(x = UMAP1, y = UMAP2, 
+      shape = seq_center, 
+      fill = broad_histology_display, 
+      color = seq_center, 
+      alpha = seq_center) + 
+  geom_point() + 
+  scale_shape_manual(values = c(21, 22, 24)) + 
+  scale_fill_manual(values  = pal_bh) +
+  scale_color_manual(values = c("grey70", "black", "black")) +
+  scale_alpha_manual(values = c(0.6, 0.8, 0.8)) +
+  ggpubr::theme_pubr() + 
+  guides(fill = FALSE, alpha = FALSE)
+```
+
+
+
+
+### Cancer group UMAP
+
+We can make this _even more_ finer-grained by exploring the distributions of cancer groups within this UMAP.
+Shapes denote the sequencing center, and colors denote the cancer group.
+
+```{r}
+ggplot(stranded_df) + 
+  aes(x = UMAP1, y = UMAP2, 
+      shape = seq_center, 
+      fill = cancer_group_display, 
+      color = seq_center, 
+      alpha = seq_center) + 
+  geom_point() + 
+  scale_shape_manual(values = c(21, 22, 24)) + 
+  scale_fill_manual(values  = pal_cg) +
+  scale_color_manual(values = c("grey70", "black", "black")) +
+  scale_alpha_manual(values = c(0.4, 0.6, 0.6)) +
+  ggpubr::theme_pubr() + 
+  guides(fill = FALSE, alpha = FALSE)
+```
+
+
+
+We can also make this plot only for cancer groups which were processed at >1 center:
+
+```{r}
+# first, which histologies are shared, aka processed at >1 center?
+shared_cg <- multiple_centers_cg %>%
+  dplyr::filter(RNA_library == "stranded") %>%
+  dplyr::distinct(cancer_group_display) %>%
+  dplyr::pull(cancer_group_display)
+
+
+stranded_df %>%
+  dplyr::filter(cancer_group_display %in% shared_cg) %>%
+  ggplot() + 
+  aes(x = UMAP1, y = UMAP2, 
+      shape = seq_center, 
+      fill = cancer_group_display, 
```

Comment:
Here too I think it would be worth included lower level diagnoses. We end up here with a lot of "other", which is not particularly helpful when we might be trying to dive deeper and understand the data. I know that we don't have a defined palette for these diagnoses, and probably not enough visual discrimination, but I don't think that is necessarily a problem at this point, when we are just exploring the data a bit.

Reply:
I'm looking into this now with the original cancer groups, before we collapsed them into the display groups, and it seems there are N=59 raw cancer groups in stranded data. But, there are only N=9 such raw cancer groups that are sequenced in >1 center. Moreover, these cancer groups are ones that made it uncollapsed into `cancer_group_display`, so I'm not sure there's much more exploration here other than color explosions?
So maybe rather than going all the way back to original cancer groups, just "uncollapse" the "Other" grouping. I'll look into this next.